### DISCUSSION TITLE
SaaS apps and Multi-tenant apps

### DISCUSSION THREAD

**Discussion by 3adel-bassiony** [adonisjs/core#3764] [Help]
Hello I have a general question, Does Adonisjs have or support any package like this to build a multi-tenant app, I used this package before to make a SaaS app using it and Laravel, so I was looking for something like this but for Adonis.js?

https://tenancyforlaravel.com/

**Comment by WeyIin** [comment]
Check this one out https://github.com/Salman-Kashfy/Multitenancy-AdonisJS-5/

**Reply by 3adel-bassiony** [reply]
@WeyIin Thank you, I will check it 💪
